ports/ace/portfile.cmake

```
@@ -1,181 +1,315 @@
+
 include(vcpkg_common_functions)
 
-# Don't change to vcpkg_from_github! This points to a release and not an archive
+set(ACE_ROOT ${CURRENT_BUILDTREES_DIR}/src/ACE_wrappers)
```

```suggestion
set(ACE_ROOT ${CURRENT_BUILDTREES_DIR}/src/ACE_wrappers)
```

https://github.com/DOCGroup/ACE_TAO/blob/57c52c1183d8d5e11e725fc9886263dfca11f22c/azure-pipelines.yml#L2

? MPC_ROOT
https://github.com/DOCGroup/ACE_TAO/blob/57c52c1183d8d5e11e725fc9886263dfca11f22c/azure-pipelines.yml#L4

---

@Voskrese 

**ACE_ROOT & MPC_ROOT**
The folder structure of the ACE_TAO git repository is different from the one in the archive. 

**default.features**
Yes. I will change that. :-)

---

Has this requirement been resolved?